packages/box/lib/utils/index.ts

```diff
@@ -40,17 +33,12 @@ export = {
     const options = {
       unsafeCleanup: true
     };
-    try {
-      const tmpDir = tmp.dirSync(options);
-      events.emit("unbox:preparingToDownload:succeed");
-      return {
-        path: path.join(tmpDir.name, "box"),
-        cleanupCallback: tmpDir.removeCallback
-      };
-    } catch (error) {
-      events.emit("unbox:fail");
-      throw error;
-    }
+    const tmpDir = tmp.dirSync(options);
+    events.emit("unbox:preparingToDownload:succeed");
+    return {
+      path: path.join(tmpDir.name, "box"),
+      cleanupCallback: tmpDir.removeCallback
+    };
```

Comment:
Ah this makes sense, I didn't realize `emit` is called in `Box` as well. Good catch!